pkg/config/templateComponent.go

```diff
@@ -122,6 +122,9 @@ func (t *TemplateComponent) Props() map[string]TemplateProperty {
 }
 
 func (t *TemplateComponent) ComponentName() string {
+	if t.CollName == "nop" {
+		return t.CollName + "/" + t.Name
```

Comment:
The `t.hpsf.GetSafeName()` was panicking on the Nop components, so I special-cased it for now as a time-boxed effort.